backend/src/bundles/users/user.controller.ts

```diff
@@ -141,6 +160,23 @@ class UserController extends Controller {
                   };
         }
     }
+
+    private async updateUserProfileAndEmail(
+        options: ApiHandlerOptions<{
+            body: UpdateUserProfileAndEmailRequestDto;
+            params: {
+                id: string;
+            };
+        }>,
+    ): Promise<ApiHandlerResponse<UserWithProfileRelation>> {
+        return {
+            status: HttpCode.OK,
+            payload: await this.userService.updateUserProfileAndEmail(
+                options.params.id,
+                options.body,
```

Comment:
Put `try catch ` block here, and consider moving the service call into a variable

Reply:
done